Source/WebCore/css/CSSComputedStyleDeclaration.cpp

```diff
@@ -4126,7 +4126,10 @@ RefPtr<CSSValue> ComputedStyleExtractor::valueForPropertyInStyle(const RenderSty
             return CSSPrimitiveValue::create(style.scrollSnapStop());
         case CSSPropertyScrollSnapType:
             return valueForScrollSnapType(style.scrollSnapType());
-
+        case CSSPropertyOverflowAnchor:
+            if (!m_element->document().settings().cssScrollAnchoringEnabled())
+                return nullptr;
+            return cssValuePool.createValue(style.overflowAnchor());
```

Comment:
```suggestion
        case CSSPropertyOverflowAnchor:
            return cssValuePool.createValue(style.overflowAnchor());
```

Since the property will never be parsed with the flag disabled, the style will never be computed.

Reply:
Fixed.